nss-day-cohort-60/national-parks-django-api
# Jd blog search

fixed blog create, update, added sorting by date for all blog GETs

To test:
1. pull JD-blog-search branch, switch, 
2. pull JD-blogs-features client branch, switch
3. navigate to blogs page
4. click add blog, add a blog
5. verify new blog renders at top of list
6. navigate to My Blogs page
7. Edit a blog, check all submitted edits are made
